tests/e2e/errlogger/errlogger.go

```
@@ -0,0 +1,191 @@
+	LogStreamCmd       *exec.Cmd
+	LogStreamWaitGroup *sync.WaitGroup
+	PodName            string
+	Stderr             io.ReadCloser
+	Stdout             io.ReadCloser
+}
+
+func (l *LogStream) ConnectStderr() {
+	GinkgoHelper()
+	stderr, err := l.LogStreamCmd.StderrPipe()
+	Expect(err).NotTo(HaveOccurred(), "failed to obtain the `Virtualization-controller` STDERR stream: %s", l.PodName)
+	l.Stderr = stderr
+}
+
+func (l *LogStream) ConnectStdout() {
+	GinkgoHelper()
+	stdout, err := l.LogStreamCmd.StdoutPipe()
+	Expect(err).NotTo(HaveOccurred(), "failed to obtain the `Virtualization-controller` STDOUT stream: %s", l.PodName)
+	l.Stdout = stdout
+}
+
+func (l *LogStream) ParseStderr() {
+	GinkgoHelper()
+	defer GinkgoRecover()
+	defer l.LogStreamWaitGroup.Done()
+
+	scanner := bufio.NewScanner(l.Stderr)
+	for scanner.Scan() {
+		GinkgoWriter.Write([]byte(fmt.Sprintf("%s%s%s\n", Red, scanner.Text(), Reset)))
+	}
+	parseScanError(scanner.Err(), "STDERR")
+}
+
+func (l *LogStream) ParseStdout(exludedPatterns []string, startTime time.Time) {
+	GinkgoHelper()
+	defer GinkgoRecover()
+	defer l.LogStreamWaitGroup.Done()
+
+	errFlag := false
+	scanner := bufio.NewScanner(l.Stdout)
+	for scanner.Scan() {
+		var entry LogEntry
+		rawEntry := strings.TrimPrefix(scanner.Text(), "0")
+		err := json.Unmarshal([]byte(rawEntry), &entry)
+		Expect(err).NotTo(HaveOccurred(), "error parsing JSON")
+		if entry.Level == LevelError && !filterMessage(rawEntry, exludedPatterns) {
+			errTime, err := time.Parse(time.RFC3339, entry.Time)
+			Expect(err).NotTo(HaveOccurred(), "failed to parse error timestamp")
+			if errTime.After(startTime) {
+				errFlag = true
+				jsonData, err := json.MarshalIndent(entry, "", "  ")
+				Expect(err).NotTo(HaveOccurred(), "error converting to JSON")
+				msg := formatMessage(
+					"this is the `Virtualization-controller` error! not the current `Ginkgo` context error:",
+					string(jsonData),
+					Red,
+				)
+				GinkgoWriter.Write([]byte(msg))
```

Do not ignore errors

---

Fixed.